recipe/conda_forge_ci_setup/upload_or_check_non_existence.py

```diff
@@ -101,7 +101,7 @@ def built_distribution_already_exists(cli, name, version, fname, owner):
 
 
 def upload(token_fn, path, owner, channels, private_upload=False):
-    cmd = ['anaconda', '--quiet', '-t', token_fn,
+    cmd = ['anaconda', '--quiet', ' --show-traceback', '-t', token_fn,
```

Comment:
A space crept in 😞

```suggestion
    cmd = ['anaconda', '--quiet', '--show-traceback', '-t', token_fn,
```

Reply:
Fixing in PR ( https://github.com/conda-forge/conda-forge-ci-setup-feedstock/pull/220 )

Reply:
FWIW looks like have the space does cause an error on upload (as seen [in CI]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=621685&view=logs&jobId=2ed0550f-e734-5ebf-52c6-1bfc19f5811c&j=80cb4bd3-7e0a-50cc-19d9-637946946f1a&t=30ab638b-e08e-5432-ccae-d862fb1f47b6 ) and [this attached log]( https://github.com/conda-forge/conda-forge-ci-setup-feedstock/files/10136821/conda-forge-ci-setup-feedstock_20221201.6_linux_64_c_compiler_version10cuda_compiler_versionNonepython3.9.____cpython.txt )). On the bright side that kept the package from publishing (so feedstocks won't see this issue).